menu.py

```diff
@@ -0,0 +1,160 @@
+"""
+	This is the Offline menu. 
+    It has a single player, two players AI mode and back options.
+    You can select an option by double clicking on it.
+"""
+import pygame as pg
+from ai import ai_move, ai_reset
+from main import main
+import random
+import sys
+
+# Game Initialization
+pg.init()
+
+# menu Resolution
+screen_width=600
+screen_height=600
+screen=pg.display.set_mode((screen_width, screen_height))
+font = pg.font.SysFont("Helvetica",75)
+#
+def display_main(choice):
+	"""
+		Link option To main board
+	"""
+	return main(choice)
+
+
+def player_format(message, playerSize, playerColor):
```

Comment:
coming from java, I can understand your proclivity to using camel case, however to conform to the project's coding style which uses a more modern python convention, pls use underscores "_" to separate variable names: so playerSize would be player_size